go-controller/pkg/ovn/acl.go

```diff
@@ -50,15 +54,43 @@ func policyTypeToAclType(policyType knet.PolicyType) aclType {
 	}
 }
 
+func policyTypeStrToAclType(policyType string) aclType {
+	switch policyType {
+	case string(knet.PolicyTypeEgress):
+		return lportEgressAfterLB
+	case string(knet.PolicyTypeIngress):
+		return lportIngress
+	default:
+		panic(fmt.Sprintf("Failed to convert PolicyType to aclType: unknown policyType type %s", policyType))
+	}
+}
+
 // hash the provided input to make it a valid portGroup name.
 func hashedPortGroup(s string) string {
 	return util.HashForOVN(s)
 }
 
+// acl.Name is cropped to 64 symbols and is used for logging.
+// this function will create the name based on DbObjectIDs, replacing all owner info with just the 2-letter
+// type of owner object.
+func getACLName(dbIDs *libovsdbops.DbObjectIDs) string {
+	t := dbIDs.GetIDsType()
+	aclPrefix := ""
+	switch {
+	case t.IsSameType(libovsdbops.ACLEgressFirewall):
+		aclPrefix = "EF"
+	case t.IsSameType(libovsdbops.ACLNetpolNamespace) || t.IsSameType(libovsdbops.ACLNetworkPolicy) || t.IsSameType(libovsdbops.ACLNetpolNode):
+		aclPrefix = "NP"
+	case t.IsSameType(libovsdbops.ACLMulticastGlobal) || t.IsSameType(libovsdbops.ACLMulticastNamespace):
+		aclPrefix = "MC"
+	}
+	return fmt.Sprintf("%.63s", aclPrefix+":"+dbIDs.StringNoOwner())
```

Comment:
I updated this function abit, it will return empty name for acls that don't use logging. So I removes the dbIDs part of it, and now it is only needed to make sure the most important information is in the name, knowing that it may be cropped (so I think no need to generalize it for now)